src/Campaigns/Actions/LoadCampaignDetailsAssets.php

```diff
@@ -32,6 +37,12 @@ public function __invoke()
                 'adminUrl' => admin_url(),
                 'currency' => give_get_currency(),
                 'isRecurringEnabled' => defined('GIVE_RECURRING_VERSION') ? GIVE_RECURRING_VERSION : null,
+                'campaignForms' => array_map(function ($form) {
```

Comment:
This could probably be replaced with an entity, but I need to figure out how to fetch the correct endpoint for a nested resource, ie `campaigns/{id}/forms`.